serialization.rst

```diff
@@ -86,6 +99,13 @@ Examples
     }
   } => { 'foo' : 42, 'prot' : "wine" }
 
+  ThatArrayClass implements MongoDB\BSON\ArraySerializable {
+    private $elements => [ 1, 2, 3 ],
+    function bsonSerialize() : array {
+        return $this->elements;
+    }
+  } => [ 1, 2, 3 ]
```

Comment:
This example conflicts with "can not be a top-level BSON document" point in line 60/61, which is what your example does here.